fold-and-r.csl

```diff
@@ -119,8 +119,8 @@
         </else-if>
         <else-if type="article-journal">
           <group delimiter=" ">
-            <text macro="title" prefix="'" suffix="'"/>
-            <text term="in" suffix=" "/>
+            <text macro="title" quotes="true"/>
+            <text term="in" prefix=", " suffix=" "/>
```

Comment:
The problem with your current version is the space it puts _before_ the comma, so that's clearly wrong.
If you just place everything in one group with ", " as the delimiter, you'll also get a comma between "in" and the journal title --> also wrong.
So I give you two options: either you place everything in one group with delimiter=" " and put a comma in a suffix on title (that's what I have four posts above) or you nest two groups with different delimiters (two posts above). Both produce the same output in most cases, but the second option is more elegant and will work sensibly even in the absence of a journal title (a very rare case, presumably).